spec/features/service_drafts/service_draft_permissions_spec.rb

```diff
@@ -0,0 +1,199 @@
+        expect(page).to have_no_content('Publish Service Draft')
+        expect(page).to have_no_content('Delete Service Draft')
+        expect(page).to have_no_content('Metadata Fields')
+        expect(page).to have_no_content('Service Information')
+        expect(page).to have_no_content('Dimensions')
+      end
+
+      it 'displays a banner message to change provider the Not Current Provider content' do
+        expect(page).to have_css('.eui-banner--warn')
+        within '.eui-banner--warn' do
+          expect(page).to have_content('You need to change your current provider to show this Service Draft')
+        end
+
+        expect(page).to have_content('Not Current Provider')
+        expect(page).to have_content('It appears you need to change your current provider to access to this content.')
+      end
+
+      context 'when clicking on warning banner link' do
+        before do
+          click_on 'You need to change your current provider to show this Service Draft'
+
+          wait_for_ajax
+        end
+
+        it 'switches the provider context' do
+          expect(User.first.provider_id).to eq('MMT_2')
+        end
+
+        it 'displays the service draft show page' do
+          within '.eui-breadcrumbs' do
+            expect(page).to have_content('Service Drafts')
+            expect(page).to have_content("#{name}")
+          end
+
+          within 'main header' do
+            expect(page).to have_css('h2', text: "#{name}")
+            expect(page).to have_content(long_name)
+          end
+        end
+
+        it 'displays the service draft preview information' do
+          within '.eui-breadcrumbs' do
+            expect(page).to have_content('Service Drafts')
+            expect(page).to have_content('Draft Title')
+          end
+
+          expect(page).to have_content(name)
+          expect(page).to have_content('Publish Service Draft')
+          expect(page).to have_content('Delete Service Draft')
+        end
+      end
+    end
+
+    context 'when trying to visit the edit draft service information page directly' do
+      before do
+        visit edit_service_draft_path(draft, 'service_information')
+      end
+
+      it 'displays the service draft show page' do
+        within '.eui-breadcrumbs' do
+          expect(page).to have_content('Service Drafts')
+          expect(page).to have_content("#{name}")
+        end
+
+        within 'main header' do
+          expect(page).to have_css('h2', text: "#{name}")
+          expect(page).to have_content(long_name)
+        end
+      end
+
+      it 'does not display the service draft preview' do
+        expect(page).to have_no_content('Publish Service Draft')
+        expect(page).to have_no_content('Delete Service Draft')
+        expect(page).to have_no_content('Metadata Fields')
+        expect(page).to have_no_content('Service Information')
+        expect(page).to have_no_content('Dimensions')
```

Comment:
I think Dimensions was from Variables

Reply:
Yep, I'll just remove it, making sure Service Information doesn't show up does the trick, I don't think I need to list every accordion